resource/css/skosmos.css

```diff
@@ -423,6 +423,7 @@ body {
   #main-container.vocabpage #sidebar-col, #main-container.termpage #sidebar-col {
     background-color: var(--vocab-bg);
     background-clip: content-box, padding-box;
+    min-height: 1000px;
```

Comment:
It looks OK to me (even though the concept card can be quite a bit smaller).
![Screenshot from 2024-03-07 13-23-16](https://github.com/NatLibFi/Skosmos/assets/578627/e0646cf5-f4aa-4d13-8009-9befc57f3e1d)